object_storage_clusters.go

```diff
@@ -48,6 +49,7 @@ func (c *Client) ListObjectStorageClusters(ctx context.Context, opts *ListOption
 	return response.Data, nil
 }
 
+// Deprecated: GetObjectStorageCluster is a deprecated API.
```

Comment:
nitpick:
```suggestion
nitpick:
```suggestion
// Deprecated: GetObjectStorageCluster uses a deprecated API endpoint.
```
```